src/style.css

```
@@ -1,58 +1,142 @@
 body {
-	padding: 10px;
-	min-width: 170px;
-	justify-content: center;
-	gap: 14px;
+  padding: 10px;
+  min-width: 170px;
+  justify-content: center;
+  gap: 14px;
+  font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen-Sans,
+    Ubuntu, Cantarell, Helvetica Neue, sans-serif;
 }
 
 button {
-	padding: 10px;
-	border: none;
-	border-radius: 2px;
-	color: white;
-	cursor: pointer;
-	background: #3aa757;
-}
+  padding: 9px 10px;
+  border: none;
+  border-radius: 4px;
+  cursor: pointer;
 
-.modeLabel {
-	display: none;
+  background: transparent;
+  color: #3aa757;
+  outline: 1px solid #3aa757;
+  box-sizing: border-box;
 }
 
-:is([br-mode="on"] #onModeLabel, [br-mode="off"] #offModeLabel) {
-	display: initial;
+button:active,
+button.selected {
+  background: #3aa757;
+  outline: 1px solid transparent;
+  color: white;
 }
 
 button:active {
-	transform: scale(1.05);
+  transform: scale(1.05);
```

1.05 looks a bit dramatic
```suggestion
  transform: scale(1.02);
```